src/domain/orders/details/order-line/edit.tsx

```diff
@@ -250,32 +265,31 @@ const OrderEditLine = ({
             />
           </div>
 
-          <div
-            className={clsx(
-              "flex small:space-x-2 medium:space-x-4 large:space-x-6",
-              { "!text-gray-400 pointer-events-none": isLocked }
-            )}
-          >
+          <div className="flex gap-6 items-center h-full">
             <div
               className={clsx(
-                "inter-small-regular text-gray-900 min-w-[60px] text-right",
-                {
-                  "!text-gray-400 pointer-events-none": isLocked,
-                }
+                "flex small:space-x-2 medium:space-x-4 large:space-x-6",
+                { "!text-gray-400 pointer-events-none": isLocked }
               )}
             >
-              {formatAmountWithSymbol({
-                amount: item.unit_price * item.quantity,
-                currency: currencyCode,
-                tax: item.tax_lines,
-                digits: 2,
-              })}
+              <div
+                className={clsx("text-gray-900 min-w-[60px] text-right", {
+                  "!text-gray-400 pointer-events-none": isLocked,
+                })}
+              >
+                {formatAmountWithSymbol({
+                  amount: item.unit_price * item.quantity,
+                  currency: currencyCode,
+                  tax: item.tax_lines,
+                  digits: 2,
+                })}
```

Comment:
There is something wrong with how this total is being calculated. I see taxes added twice. 

Do we have access to `item.total` which we typically respond with on the backend?

Reply:
Maybe we should add line item totals to order edit line items to make this simpler

Reply:
> Do we have access to item.total which we typically respond with on the backend

We don't, should we open a ticket for the core to decorate OE items with totals as we do with order items?

Reply:
Yes will do it :)